src/KubernetesConfiguration/README.md

```diff
@@ -83,27 +91,101 @@ directive:
           "description": "Conflict",
           "x-ms-error-response": true,
           "schema": {
-            "$ref": "https://github.com/Azure/azure-rest-api-specs/blob/fa0a95854a551be7fdb04367e2e7b6500ab2e341/specification/common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            "$ref": "https://github.com/Azure/azure-rest-api-specs/blob/791255f0c5dd775015cd51f3e642549190fb3803/specification/common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
Can we use $(repo) here so that we don't need to update here whenever the swagger is updated?
```suggestion
            "$ref": "$(repo)/specification/common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Reply:
Writing code using $(repo) causes errors when autorest is executed

Reply:
@xiaogang-ding